presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/PushPredicateIntoTableScan.java

```diff
@@ -73,13 +73,13 @@
  * These rules should not be run after AddExchanges so as not to overwrite the TableLayout
  * chosen by AddExchanges
  */
-public class PickTableLayout
+public class PushPredicateIntoTableScan
```

Comment:
I think changes in this class should be extracted to separate commit.

Reply:
Unfortunately, I don't think I can. The change is logically tied together with removing the notion of table layouts from the engine. If I make these changes in a commit before or after the current one, we're left with a build that doesn't work.